src/expm_frechet.jl

```diff
@@ -0,0 +1,532 @@
+
+@doc raw"""
+based on
+.. [1] Awad H. Al-Mohy and Nicholas J. Higham (2009)
+           Computing the Frechet Derivative of the Matrix Exponential,
+           with an application to Condition Number Estimation.
+           SIAM Journal On Matrix Analysis and Applications.,
+           30 (4). pp. 1639-1657. ISSN 1095-7162
+"""
+
+ell_table_61 = (
+    nothing,
+    # 1
+    2.11e-8,
+    3.56e-4,
+    1.08e-2,
+    6.49e-2,
+    2.00e-1,
+    4.37e-1,
+    7.83e-1,
+    1.23e0,
+    1.78e0,
+    2.42e0,
+    # 11
+    3.13e0,
+    3.90e0,
+    4.74e0,
+    5.63e0,
+    6.56e0,
+    7.52e0,
+    8.53e0,
+    9.56e0,
+    1.06e1,
+    1.17e1,
+)
+
+@doc raw"""
+    _diff_pade3(A, E)
+
+Compute expm using 3-term pade approximant
```

Comment:
hm, sorry, this might be too less information, I am not sure what this function does.

Reply:
expanded docstring
@doc raw"""
 34    _diff_pade3(A, E)
 35    Compute U, V, LU, LV from the first 6 lines of the for loop in
 36    algorithm 6.4 of [^AlMohyHigham2009] using 3-term Padé approximant
 37    the tuple b contains the Padé coefficients of the numerator
 38"""
 39@inline function _diff_pade3!(buff, A, E)
 40    b = (120.0, 60.0, 12.0, 1.0)

Reply:
line 34 needs to be indented by four letters and followed by an empty line to render correctly.

Reply:
thanks - this is fixed - BTW, this function is internal, it does not show on the doc. - see more below

Reply:
Internal or not, as a maintainer I still have to understand what it does and avoid that it has bugs, because the issues here in this repository are things _I_ have to answer and fix. So for that internal or external is not an issue, stability and clarity (and bug-free functions ;)) are the main issue.